docs/sources/datasources/postgres.md

```diff
@@ -19,20 +19,22 @@ Grafana ships with a built-in PostgreSQL data source plugin that allows you to q
 
 ### Data source options
 
-Name           | Description
--------------- | -------------
-`Name`         | The data source name. This is how you refer to the data source in panels and queries.
-`Default`      | Default data source means that it will be pre-selected for new panels.
-`Host`         | The IP address/hostname and optional port of your PostgreSQL instance. _Do not_ include the database name. The connection string for connecting to Postgres will not be correct and will cause errors.
-`Database`     | Name of your PostgreSQL database.
-`User`         | Database user's login/username
-`Password`     | Database user's password
-`SSL Mode`     | This option determines whether or with what priority a secure SSL TCP/IP connection will be negotiated with the server.
-`Max open`     | The maximum number of open connections to the database, default `unlimited` (Grafana v5.4+).
-`Max idle`     | The maximum number of connections in the idle connection pool, default `2` (Grafana v5.4+).
-`Max lifetime` | The maximum amount of time in seconds a connection may be reused, default `14400`/4 hours (Grafana v5.4+).
-`Version`      | This option determines which functions are available in the query builder (only available in Grafana 5.3+).
-`TimescaleDB`  | TimescaleDB is a time-series database built as a PostgreSQL extension. If enabled, Grafana will use `time_bucket` in the `$__timeGroup` macro and display TimescaleDB specific aggregate functions in the query builder (only available in Grafana 5.3+).
+Name               | Description
+-----------------  | -------------
+`Name`             | The data source name. This is how you refer to the data source in panels and queries.
+`Default`          | Default data source means that it will be pre-selected for new panels.
+`Host`             | The IP address/hostname and optional port of your PostgreSQL instance. _Do not_ include the database name. The connection string for connecting to Postgres will not be correct and will cause errors.
```

Comment:
```suggestion
`Host`             | The IP address/hostname and optional port of your PostgreSQL instance. _Do not_ include the database name. The connection string for connecting to Postgres will not be correct and it may cause errors.
```